packages/core/tests/unit/models/bard-request-v2/request-test.js

```diff
@@ -25,6 +26,7 @@ module('Unit | Model | Fragment | BardRequest  - Request', function(hooks) {
                     field: 'dateTime',
                     operator: 'bet',
                     type: 'timeDimension',
+                    parameters: { grain: 'day' },
```

Comment:
We'll see how this turns out. I'm thinking we can keep this as a parameter here and configure it strictly in the metadata so that it will work either way